### PR TITLE
feat(packages/atst): Final touches on the docs

### PR DESCRIPTION
**Pull request by qbzzt**
Closing: FE-1255, FE-1262

### CODE REVIEW

File: packages/atst/README.md [hunk 1]
```diff
@@ -28,42 +28,50 @@ Install
 npm install @eth-optimism/atst wagmi @wagmi/core ethers@5.7.0
 ```
 
-## atst typescript sdk
+## atst typescript/javascript sdk
```
Comment (roninjin10) [inline packages/atst/README.md:31]: nit redundant could jut call it a typescript sdk or a javascript sdk

File: packages/atst/docs/sdk.md [hunk 2]
```diff
@@ -265,24 +263,25 @@ const bigNumberAttestation = stringifyAttestationBytes(
...
+- `useAttestationStationAttest` - Wraps useContractWrite with AttestationStation abi calling attest
+- `usePrepareAttestationStationAttest` - Wraps usePrepare with AttestationStation abi calling attest
 - `useAttestationStationAttestationCreatedEvent` - Wraps useContractEvents for Created events
 
 
 
 ## Tutorial
 
-For a tutorial on using the attestation station in general, see out tutorial as well as other Optimism related tutorials in our [optimism-tutorial](https://github.com/ethereum-optimism/optimism-tutorial/tree/main/ecosystem/attestation-station#key-values) repo.
+For a tutorial on using AttestationStation in general, see our tutorial as well as other Optimism related tutorials in our [optimism-tutorial](https://github.com/ethereum-optimism/optimism-tutorial/tree/main/ecosystem/attestation-station) repo.
```
Comment (roninjin10) [inline packages/atst/docs/sdk.md:286]: If you wanted you could link to the optimism starter in the tutorial as an example of using attestation station with react

**Review by roninjin10 — APPROVED** [review]

File: packages/atst/package.json [hunk 3]
```diff
@@ -59,5 +59,14 @@
...
-  }
+  },
+  "keywords": [
+    "react",
+    "hooks",
+    "eth",
+    "ethereum",
+    "dapps",
+    "web3",
+    "attestation"
```
Comment (roninjin10) [inline packages/atst/package.json:70]: Maybe optimism too

File: packages/atst/README.md (hunk 1 above)
Comment (qbzzt) [reply packages/atst/README.md:31]: @zainbacchus asked for it to be both.

File: packages/atst/docs/sdk.md (hunk 2 above)
Comment (qbzzt) [reply packages/atst/docs/sdk.md:286]: Good point. I'll add that. I'll also squash the commits, since I'll need to do that anyway.

**Pull request merged**